src/kbmod/filters/stamp_filters.py

```diff
@@ -0,0 +1,68 @@
+import abc
```

Comment:
This file has three different wordings for the filter:

- `stamp_filter` (the file name)
- `StampPeakFilter` (class name)
- `PeakOffset` (filter name)

Can we converge on something?

Reply:
The file name is meant to be more general than the class name since I am planning on having a few basic stamp filters in this file.

I changed the filter name to match the class name though.